### ISSUE TITLE
Implement a less.js compatibility test suite

### ISSUE THREAD

**Issue by rytmis** [dotless/dotless#450]
Less.js has what looks to me like a fairly extensive set of tests that verify its behavior. Happily, it's written as a bunch of less files along with the expected CSS output, so porting the test suite should be fairly straightforward and give us a pretty clear roadmap towards full feature parity.

**Comment by lukeapage** [comment]
Yep, the files even used to match.. About less.js 1.3.x was when i stopped syncing between them.

**Comment by rytmis** [comment]
Sorry, I didn't quite get that -- syncing between what exactly? The LESS and CSS file names?

**Comment by lukeapage** [comment]
Between less.js and dotless tests
